utils/build/docker/set-system-tests-weblog-env.Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV DD_ENV=system-tests
 ENV DD_APPSEC_ENABLED=true
 ENV DD_TRACE_DEBUG=true
 ENV DD_TRACE_LOG_DIRECTORY=/var/log/system-tests
+ENV DD_TRACE_COMPUTE_STATS=true
```

Comment:
we're likely changing/establishing this to be `DD_TRACE_STATS_COMPUTATION_ENABLED`

but it's ok as-is for now 👍

Reply:
ok, please let me know when you change it :)